pkg/resource/aws/deserializer/sns_topic_subscription_deserializer.go

```diff
@@ -23,6 +23,7 @@ func (s SNSTopicSubscriptionDeserializer) Deserialize(subscriptionsList []cty.Va
 	subscriptions := make([]resource.Resource, 0)
 
 	for _, value := range subscriptionsList {
+
```

Comment:
Why this new line ?

Reply:
nice catch, I missed this one as I used to check "ignore whitespaces changes" in github UI

Reply:
@moadibfr